### PR TITLE
Restructure Book

### PR DESCRIPTION
**Pull request by acocac**
The PR creates sections and content for Part 1, 2 and 3 of the Pangeo/OpenEO workshops.

### CODE REVIEW

**Comment by clausmichele** [comment]
Thansk @acocac, we will add what's missing in Part 2 from the openEO side.

**Comment by acocac** [comment]
@clausmichele would you prefer to work on the missing part 2 in a separate branch? If this is ok, we'll merge the PR after @pl-marasco @annefou approvals. Then you can add content and open a PR (feel free to tag any of us for a review).

**Comment by clausmichele** [comment]
> @clausmichele would you prefer to work on the missing part 2 in a separate branch? If this is ok, we'll merge the PR after @pl-marasco @annefou approvals. Then you can add content and open a PR (feel free to tag any of us for a review).

Yes, we can open a separate PR for that part.

File: tutorial/part1/overview.md
Comment (annefou) [inline tutorial/part1/overview.md:1]: Maybe we can reuse markdown from what we have at https://github.com/galaxyproject/training-material/blob/main/topics/climate/tutorials/pangeo-notebook/slides.html (and split motivation & overview)

**Review by annefou — APPROVED** [review]

**Pull request merged**